app/controllers/api/comments_controller.rb

```diff
@@ -0,0 +1,28 @@
+module Api
+  class Api::CommentsController < ApiController
+    # load_and_authorize_resource
```

Comment:
**You both did great job!**
I like the API implementations 👍 

- [ ] However, this line 3 looks odd. If this line of code is not necessary kindly remove it.

Reply:
thanks! It's done